modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
I agree with @paweldomas here. We could call it `_sdpTransform`, so when you do this:

```
const desc = new RTCSessionDescription({type: 'offer', _sdpTransform: theSdpTransformThing});
```

it picks it up.

I guess you are also in favor of dropping the `parsed` getter then @paweldomas ? The Edge shim can read `_parsed` because it's an internal thing, and that's ok.

Now, "parsed" is a bad name, it leads to a bit of confusion... `_sdpObj`? `_sdpTransform` ?